html/pages/deluser.inc.php

```diff
@@ -14,7 +14,7 @@
             $delete_username = dbFetchCell('SELECT username FROM users WHERE user_id = ?', array($vars['id']));
 
             if ($vars['confirm'] == 'yes') {
-                if (deluser($delete_username)) {
+                if (deluser($vars['id']) >= 0) {
```

Comment:
That function is in all of the auth modules so it would mean updating them all to do that same check and return true / false when we can just do it here.